src/audio/module_adapter/module_adapter.c

```diff
@@ -828,19 +828,21 @@ static int module_adapter_simple_copy(struct comp_dev *dev)
 int module_adapter_copy(struct comp_dev *dev)
 {
 	struct processing_module *mod = comp_get_drvdata(dev);
-	struct module_data *md = &mod->priv;
+	struct module_data *md;
 	struct comp_buffer *source, *sink;
-	struct comp_buffer __sparse_cache *sink_c = NULL;
+	struct comp_buffer __sparse_cache *sink_c;
 	struct list_item *blist;
-	size_t size = MAX(mod->deep_buff_bytes, mod->period_bytes);
-	uint32_t min_free_frames = UINT_MAX;
-	int ret, i = 0;
+	size_t size;
+	uint32_t min_free_frames;
+	int ret, i;
 
 	comp_dbg(dev, "module_adapter_copy(): start");
 
 	if (mod->simple_copy)
 		return module_adapter_simple_copy(dev);
```

Comment:
then module_adapter_simple_copy can be static inline, this can save one more 0.1